browser/resources/settings/brave_overrides/appearance_page.ts

```diff
@@ -38,6 +38,13 @@ RegisterPolymerTemplateModifications({
         `)
     }
 
+    const customizeToolbar = templateContent.getElementById('customizeToolbar')
```

Comment:
Q) Is this visible in cr135? I can't see in our appearance page.

Reply:
I am definitely seeing it: 
![image](https://github.com/user-attachments/assets/1d609d0f-44a4-4371-b7c1-6fd9b18beb51)

Reply:
Yep, me too.

Reply:
It's different on macOS and Win11. I also can see it on Win11 but it's not visible on macOS.